documentation/src/main/asciidoc/userguide/appendices/LegacyBasicTypeResolution.adoc

```diff
@@ -0,0 +1,346 @@
+|LocalTimeType |TIME |java.time.LocalTime |LocalTime, java.time.LocalTime
+|OffsetDateTimeType |TIMESTAMP |java.time.OffsetDateTime |OffsetDateTime, java.time.OffsetDateTime
+|OffsetTimeType |TIME |java.time.OffsetTime |OffsetTime, java.time.OffsetTime
+|ZonedDateTimeType |TIMESTAMP |java.time.ZonedDateTime |ZonedDateTime, java.time.ZonedDateTime
+|=================================================================================================
+
+.Hibernate Spatial BasicTypes
+[cols="<.^,<.^,<.^,<.^",options="header",]
+|=================================================================================================
+|Hibernate type (org.hibernate.spatial package) |JDBC type |Java type |BasicTypeRegistry key(s)
+|JTSGeometryType |depends on the dialect | com.vividsolutions.jts.geom.Geometry |jts_geometry, and the class names of Geometry and its subclasses
+|GeolatteGeometryType |depends on the dialect | org.geolatte.geom.Geometry |geolatte_geometry, and the class names of Geometry and its subclasses
+|=================================================================================================
+
+[NOTE]
+====
+To use the Hibernate Spatial types, you must add the `hibernate-spatial` dependency to your classpath _and_ use an `org.hibernate.spatial.SpatialDialect` implementation.
+
+See the <<chapters/query/spatial/Spatial.adoc#spatial,Spatial>> chapter for more details.
+====
+
+
+
+[[basic-registry]]
+=== BasicTypeRegistry
+
+We said before that a Hibernate type is not a Java type, nor an SQL type, but that it understands both and performs the marshalling between them.
+But looking at the basic type mappings from the previous examples,
+how did Hibernate know to use its `org.hibernate.type.StringType` for mapping for `java.lang.String` attributes,
+or its `org.hibernate.type.IntegerType` for mapping `java.lang.Integer` attributes?
+
+The answer lies in a service inside Hibernate called the `org.hibernate.type.BasicTypeRegistry`, which essentially maintains a map of `org.hibernate.type.BasicType` (an `org.hibernate.type.Type` specialization) instances keyed by a name.
+
+We will see later, in the <<basic-type-annotation>> section, that we can explicitly tell Hibernate which BasicType to use for a particular attribute.
+But first, let's explore how implicit resolution works and how applications can adjust the implicit resolution.
+
+[NOTE]
+====
+A thorough discussion of `BasicTypeRegistry` and all the different ways to contribute types is beyond the scope of this documentation.
+
+Please see the http://docs.jboss.org/hibernate/orm/{majorMinorVersion}/integrationguide/html_single/Hibernate_Integration_Guide.html[Integration Guide] for complete details.
+====
+
+As an example, take a String attribute such as we saw before with Product#sku.
+Since there was no explicit type mapping, Hibernate looks to the `BasicTypeRegistry` to find the registered mapping for `java.lang.String`.
```

Comment:
Mix of tenses. Change "was" to "is" or "looks" to "looked".

Reply:
The second sentence, yeah I agree - it should be present tense.  

The first should absolutely be past tense though.